Explain why multiple connection IDs (#4586)
quicwg/base-drafts: draft-ietf-quic-transport.md

be routed back to the endpoint and to be identified by the endpoint upon
receipt.

Multiple connection IDs are used so that endpoints can send packets that cannot

> be identified as being for the same connection without cooperation from an
> endpoint; see {{migration-linkability}}.

Reviewer: ```suggestion
be identified as being for the same connection without cooperation from their
peer; see {{migration-linkability}}.
```

Author: This is talking about any entity doing the linking, and either endpoint in the connection can provide them with the ability to correlate packets for the same connection.  What I proposed is, I believe, correct.

> be identified as being for the same connection without cooperation from an
> endpoint; see {{migration-linkability}}.

Reviewer 2: ```suggestion
be identified by an observer as being for the same connection without
cooperation from an endpoint; see {{migration-linkability}}.
```
